public/js/components/Charts/ForkTimeCharts.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Themes } from 'formidable-charts';
+import { Grid, Row, Col } from 'react-flexbox-grid';
+import AreaChartWrap from './AreaChartWrap';
+import ScatterChartWrap from './ScatterChartWrap';
+import { CHART_FILTERS_MAP } from "../../utils/chartList";
+
+const NoDataMessage = ({ filters }) =>
+    <div className="no-data">
+        No data, please update any of the following:<br />
+        <span className="filters">{filters.join(", ")}</span>
+    </div>
+
+const ForkTimeCharts = ({ charts, isUpdating, filterVals }) => {
```

Comment:
This is repeated in `OriginCharts.js` I wasn't sure the best way to pull it out.